models/src/main/scala/coop/rchain/models/rholang/sorter/GroundSortMatcher.scala

```diff
@@ -17,59 +17,6 @@ private[sorter] object GroundSortMatcher extends Sortable[ExprInstance] {
       case gi: GInt    => ScoredTerm(g, Leaves(Score.INT, gi.value)).pure[F]
       case gs: GString => ScoredTerm(g, Node(Score.STRING, Leaf(gs.value))).pure[F]
       case gu: GUri    => ScoredTerm(g, Node(Score.URI, Leaf(gu.value))).pure[F]
-      case EListBody(gl) =>
-        for {
-          sortedPars <- gl.ps.toList.traverse(Sortable[Par].sortMatch[F])
-        } yield
-          ScoredTerm(
-            EListBody(gl.withPs(sortedPars.map(_.term.get))),
-            Node(Score.ELIST, sortedPars.map(_.score): _*)
-          )
-      case ETupleBody(gt) =>
-        for {
-          sortedPars <- gt.ps.toList.traverse(Sortable[Par].sortMatch[F])
-        } yield
-          ScoredTerm(
-            ETupleBody(gt.withPs(sortedPars.map(_.term.get))),
-            Node(Score.ETUPLE, sortedPars.map(_.score): _*)
-          )
-      // Note ESet and EMap rely on the stableness of Scala's sort
-      // See https://github.com/scala/scala/blob/2.11.x/src/library/scala/collection/SeqLike.scala#L627
-      case ESetBody(gs) =>
-        for {
-          pars              <- gs.ps.sortedPars.traverse(Sortable[Par].sortMatch[F])
-          sortedPars        = pars.sorted
-          remainderScoreOpt = gs.remainder.map(_ => Leaf(Score.REMAINDER))
-        } yield
-          ScoredTerm(
-            ESetBody(
-              ParSet(
-                SortedParHashSet(sortedPars.map(_.term.get)),
-                gs.connectiveUsed,
-                gs.locallyFree,
-                gs.remainder
-              )
-            ),
-            Node(Leaf(Score.ESET) :: sortedPars.map(_.score) ::: remainderScoreOpt.toList)
-          )
-      case EMapBody(gm) =>
-        def sortKeyValuePair(key: Par, value: Par): F[ScoredTerm[(Par, Par)]] =
-          for {
-            sortedKey   <- Sortable.sortMatch(key)
-            sortedValue <- Sortable.sortMatch(value)
-          } yield ScoredTerm((sortedKey.term, sortedValue.term), sortedKey.score)
-
-        for {
-          pars              <- gm.ps.sortedList.traverse(kv => sortKeyValuePair(kv._1, kv._2))
-          sortedPars        = pars.sorted
-          remainderScoreOpt = gm.remainder.map(_ => Leaf(Score.REMAINDER))
-        } yield
-          ScoredTerm(
-            EMapBody(
-              ParMap(sortedPars.map(_.term), gm.connectiveUsed, gm.locallyFree, gm.remainder)
-            ),
-            Node(Leaf(Score.EMAP) :: sortedPars.map(_.score) ::: remainderScoreOpt.toList)
-          )
```

Comment:
You can just delete all this?

Reply:
Oh nvm, I see

Reply:
We do all of them in `ExprSortMatcher.scala` (where I think they belong).

Reply:
I guess they ended up in both places because I added collection sorting to `ExprSortMatcher.scala` since I wasn't aware they were already in `GroundSortMatcher.scala`...